javascript/javascript1/week1/ezName.js

```diff
@@ -0,0 +1,9 @@
+
+let firstWords = ["Easy", "Awesome", "Wonderful","Ideal","Professional","Creative","Hub","Solution"];
+let secondWords = ["Feasible", "Valuable", "Pricy","Incredible","Hoody","Easy", "Awesome", "Wonderful"];
+const num1 = 2;
+const num2 = 2;
```

Comment:
you can use the random number generator mentioned at the homework instead of both `num1`, `num2`
```js
const randomNumber = Math.floor(Math.random() * 10) + 0;
```